Source/EventFlow/Snapshots/SnapshotAggregateRoot.cs

```diff
@@ -69,18 +69,14 @@ public abstract class SnapshotAggregateRoot<TAggregate, TIdentity, TSnapshot> :
 
             await LoadSnapshotContainerAsync(snapshot, cancellationToken).ConfigureAwait(false);
 
+            var version = snapshot.Metadata.AggregateSequenceNumber;
```

Comment:
By simply writing

```csharp
Version = snapshot.Metadata.AggregateSequenceNumber;
```

wouldn't that ensure that events versions are correct during apply after a snapshot load?

Reply:
Yes. Fixed!